website/versioned_docs/version-1.1/plugins/optuna_sweeper.md

```diff
@@ -41,11 +40,11 @@ defaults:
 
 Alternatively, add `hydra/sweeper=optuna` option to your command line.
 
-The default configuration is <GithubLink to="plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py">here</GithubLink>.
+The default configuration is <GithubLink to="plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py">here</GithubLink>. # Omry: broken link
```

Comment:
this is not a broken link (you can try clicking it in our documentation. ) GithubLink is a component.